### ISSUE TITLE
GuardManager Factories doesnt work

### ISSUE THREAD

**Issue by FabianKoestring** [LM-Commons/LmcRbacMvc#138]
### Bug Report

| Q          | A     |
|------------|-------|
| Version(s) | 4.0.0 |

#### Summary
In older versions we used following code (as shown in the [docs](https://lm-commons.github.io/LmcRbacMvc/docs/guards#creating-custom-guards)) to create a custom guard.

```php
'guard_manager' => [
    'factories' => [
        Guard\MyGuard::class => Guard\MyGuardFactory::class
    ]
],
```
In current version `4.0.0` it seems like the factory isn´t called anymore.

**Comment by visto9259** [comment]
Let me check into this.

**Comment by visto9259** [comment]
Yes.  There is an issue here. 
I will fix it later today.

**Comment by visto9259** [comment]
In the refactoring, the GuardPluginManager factory was no longer using the `'guard_manager'` config key.
This was a stupid oversight.
 `'guard_manager'` is now a module option in `Lmc\Rbac\Mvc\Options\ModuleOptions` and the Gaurd Plugin Manager factory uses this new option.
Thanks @FabianKoestring for reporting this issue.
Fixed in 4.0.1
